src/runtime/mini_mman.h

```diff
@@ -0,0 +1,30 @@
+#ifndef HALIDE_RUNTIME_MMAN_H
```

Comment:
huh: I have apparently never needed to actually add a #include for these calls before.